testing/run_deploy_app.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+#
+# Scripts to start a load test for 1-click-deployment service.
+#
+# GOOGLE_APPLICATION_CREDNTIALS and SERVICE_CLIENT_ID both point to
```

Comment:
What is this script for?
e.g. is this for manual testing or is it intended to be run in cluster as a cron job to run the load test regularly?

Reply:
This is for manually running the load test.

Reply:
Could you update the comment please?

How come you are using a service account if you are running manually?

Why wouldn't you just use the credentials for @zhenghuiwang  configured via gcloud?

Reply:
I updated the comment.

I think the load test/ probe test are written with a cron job in mind. This script is just for me to test it actually works manually.

There are many env vars and cmd flags. So I think it is useful to document the entry point via a script.